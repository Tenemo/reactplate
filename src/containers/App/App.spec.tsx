import React from 'react';

describe('App', () => {
    it('should match the snapshot', () => {});
});
