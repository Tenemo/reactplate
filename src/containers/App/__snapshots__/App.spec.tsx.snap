// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<ThemeProvider
  theme={
    Object {
      "primary": "#222",
      "secondary": "#CCC",
    }
  }
>
  <styled.div>
    <GlobalStyleComponent />
    <Header />
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="*"
      />
    </Switch>
  </styled.div>
</ThemeProvider>
`;
